float32 value  # Double-precision floating point value of the parameter
---
